pkg/controller/compute/gke_test.go

```diff
@@ -98,7 +98,8 @@ func TestSyncClusterGetError(t *testing.T) {
 	tc := testCluster()
 
 	r := &Reconciler{
-		Client: NewFakeClient(tc),
+		Client:                   NewFakeClient(tc),
+		ManagedReferenceResolver: resource.NewAPIManagedReferenceResolver(NewFakeClient()),
```

Comment:
Would it be better to use a no-op `ManagedReferenceResolverFn` for these tests given we don't intend to test the reference resolver code?

Reply:
added a struct with noop method